_includes/package-grid.html

```diff
@@ -46,10 +46,11 @@ <h3 class="card__title no_toc" itemprop="headline">
                 <li><a href="{{ apackage.archive }}" rel="permalink"><i class="fas fa-bookmark fa-fw"></i> JOSS Approved</a></li>
             {% endif %}
             {% if apackage.astropy %}
-                <li><a href="partners/astropy.html"><i class="fa-solid fa-check-double"></i> <img src="http://img.shields.io/badge/Affiliated-Astropy-orange.svg?style=flat" alt="Astropy" /></a></li>
+                <li><a href="communities/astropy.html"><i class="fa-solid fa-check-double"></i> <img src="http://img.shields.io/badge/Affiliated-Astropy-orange.svg?style=flat" alt="Astropy" /></a></li>
```

Comment:
This communities vs partners is confusing me. I *think* this got rid of one of the Jekyll errors though but I cannot be sure.